es-core/src/components/ImageGridComponent.h

```diff
@@ -158,18 +439,76 @@ void ImageGridComponent<T>::render(const Eigen::Affine3f& parentTrans)
 		mEntriesDirty = false;
 	}
 
-	for(auto it = mImages.begin(); it != mImages.end(); it++)
-	{
-		it->render(trans);
+	int i = 0;
+	std::shared_ptr<GridTileComponent> pTile;
+	bool bSelected = false;
+	for (auto ti = mTiles.begin(); ti != mTiles.end(); ti++) {
+		if (i > getEntryCount() - 1) break;
+		// Keep the selected tile and render it later (so it draws on top others)
+		if ((*ti)->isSelected()) {
+			i++;
+			pTile = (*ti);
+			bSelected = true;
+		}
+		else {
+			i++;
+			(*ti)->render(trans);
+		}
+	}
+
+	if (bSelected) {
+		pTile->render(trans);
 	}
 
+
+	listRenderTitleOverlay(trans);
+
 	GuiComponent::renderChildren(trans);
 }
 
 template<typename T>
+void ImageGridComponent<T>::applyThemeToChildren(const std::shared_ptr<ThemeData>& theme) {
```

Comment:
yeah, so to me this look like a regular applyTheme, not the applyTheme to the children of the ImageGridComponent...

Reply:
Changed to regular applyTheme